locust/web.py

```diff
@@ -5,6 +5,7 @@
 import logging
 import mimetypes
 import os.path
+import sys
```

Comment:
I think this import was added accidentally? I wonder why it doesnt fail linting...